code/modules/reagents/reagent_dispenser.dm

```diff
@@ -329,3 +335,27 @@
 		to_chat(user, "<span class='notice'>Sprayer refilled.</span>")
 		playsound(get_turf(src), 'sound/effects/refill.ogg', 50, 1, -6)
 		return 1
+		
+/obj/structure/reagent_dispensers/proc/wrenchAnchor(var/mob/user) //proc to wrench a wrenchable dispenser into place
```

Comment:
I did, yeah
When you say move the proc to the /obj level, do you mean remove the machinery proc as well and make a general wrenchAnchor for every object?
Would that mean moving wrenchable up to that same level?(every object is either wrenchable or not)
Also, I've apparently been confusing file directories with object pathing. Where in the directory would I go to place something in the /obj level?